assets/prototype/application/utilities/money/amountsMatch.ts

```diff
@@ -11,4 +11,6 @@ const amountsMatch = (amount1: number | string, amount2: number | string): boole
 	return parsedAmount(amount1) === parsedAmount(amount2);
 };
 
+// export const amountsMatch = (v1, v2) => parseFloat(v1) === parseFloat(v2);
+
```

Comment:
Fix for: 
![image](https://user-images.githubusercontent.com/1477453/72340021-0b5d9000-36d0-11ea-8188-5f0bd3e54eac.png)